### ISSUE TITLE
+vats missing fields incl. source ship, publishing ship

### ISSUE THREAD

**Issue by sethfork** [urbit/urbit#6637]
_This may just be a documentation issue - redirect if so._

It looks like several fields were removed from the default output of `+vats`, including source ship & publishing ship, which are generally useful to see when inspecting desks over the course of normal operations, troubleshooting, developing, etc.
```hoon
> +vats %webterm
%webterm
  /sys/kelvin:           [%zuse 416] [%zuse 415] [%zuse 414] [%zuse 413]
  base hash ends in:     ~[ap8ot jdepo]
  %cz hash ends in:      jdepo
  app status:            running
  pending updates:       ~
::
```
`+vat %desk` still works properly but is getting deprecated - and neither the deprecation warning, nor the [documentation](https://operators.urbit.org/manual/os/dojo-tools#vats) mention the difference in default functionality between the two, or how to print all the expected fields. If the default output of `+vats` is being reduced, the docs &/or deprecation warning should provide that guidance. 
```hoon
> +vat %webterm
%webterm
  /sys/kelvin:      [%zuse 416] [%zuse 415] [%zuse 414] [%zuse 413]
  base hash:        ~[0v1c.r1qd1.7k3j2.1rmti.jq0tg.gpdp2.c2d58.npmcu.nndk1.j90ds.ap8ot 0v1.46831.2647j.0n6am.rfndg.mt1hp.r1n7j.cnorc.j0rf6.i1o12.jdepo]
  %cz hash:         0v1.46831.2647j.0n6am.rfndg.mt1hp.r1n7j.cnorc.j0rf6.i1o12.jdepo
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~mister-dister-dozzod-dozzod]
  updates:          remote
  source ship:      ~malbel
  source desk:      %webterm
  source aeon:      13
  kids desk:        ~
  pending updates:  ~
::
```
```hoon
Notice: +vat is deprecated. use +vats which now takes one or more desks as arguments. e.g. '+vats %base %garden'
```

**Comment by MarcusMiguel** [comment]
It's a documentation issue, right now you need to `+vats %desk, =verb &` to see the full details.

**Comment by joemfb** [comment]
This has come up pretty often -- the default should probably be a little more verbose.

**Comment by jalehman** [comment]
@joemfb you closed this in a past PR, right?

**Comment by joemfb** [comment]
Yes, fixed in #6670
